tests/dialects/test_tsql.py

```diff
@@ -17,6 +17,11 @@ def test_tsql(self):
                 "spark": "DROP TABLE IF EXISTS TempTableName",
             },
         )
+
+        self.validate_identity(
+            "MERGE INTO mytable WITH (HOLDLOCK) AS T USING mytable_merge AS S "
+            "ON (T.user_id = S.user_id) WHEN NOT MATCHED THEN INSERT (c1, c2) VALUES (S.c1, S.c2)"
+        )
```

Comment:
@dmoore247 note the addition of `VALUES (S.c1, S.c2)`. Your example statement didn't include this part, but it seems like it's required (the `VALUES` part, to be specific), based on the syntax spec.